packages/office-ui-fabric-react/src/components/Fabric/Fabric.styles.ts

```diff
@@ -16,10 +16,12 @@ export interface IFabricClassNames {
 export const getStyles = (props: IFabricStyleProps): IFabricStyles => {
   const { theme, className, isFocusVisible, applyTheme } = props;
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
+  const focusVisibility = isFocusVisible ? 'Visible' : 'Hidden';
   return {
     root: [
       classNames.root,
-      isFocusVisible && 'is-focusVisible ms-Fabric--isFocusVisible',
+      // keywords for search: is-focusVisible ms-Fabric--isFocusVisible ms-Fabric--isFocusHidden
+      `is-focus${focusVisibility} ms-Fabric--isFocus${focusVisibility}`,
```

Comment:
ok, this makes sense. instead of adding `is-focusVisible ms-Fabric--isFocusVisible` to the Fabric control when focus is visible, we toggle between:

```
is-focusVisible ms-Fabric--isFocusVisible

// and 

is-focusHidden ms-Fabric--isFocusHidden
```

Reply:
That's correct.

Reply:
...so with respect to the 'visible' class, this PR should change nothing.